BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClientParams.java

```diff
@@ -18,6 +18,7 @@ class BraintreeClientParams {
     private BrowserSwitchClient browserSwitchClient;
     private ManifestValidator manifestValidator;
     private UUIDHelper uuidHelper;
+    private String defaultReturnURLScheme;
```

Comment:
Nitpick - internal naming still is using `default`. Do we want to change this to closer mirror the new public property naming? Like `newTaskReturlURLScheme`?

Reply:
Yeah this makes sense. We may go with `braintreeDeepLinkReturnUrlScheme`. Also added a `NEXT MAJOR VERSION` note to formalize capitalization of `URL` etc. since at the moment other properties have a `Url` convention that isn't consistent.